packages/alfa-rules/test/sia-r61/rule.spec.tsx

```diff
@@ -1,13 +1,15 @@
+import { Device } from "@siteimprove/alfa-device";
+import { Document } from "@siteimprove/alfa-dom";
 import { jsx } from "@siteimprove/alfa-dom/jsx";
 import { test } from "@siteimprove/alfa-test";
 
-import { Document } from "@siteimprove/alfa-dom";
-
 import R61, { Outcomes } from "../../src/sia-r61/rule";
 
 import { evaluate } from "../common/evaluate";
 import { passed, failed, inapplicable } from "../common/outcome";
 
+const device = Device.standard();
```

Comment:
There's no need to pass the standard device parameters to `evaluate()` and this is already the default if none are passed.

Reply:
Magic!
And I'm passing it so often 🙈

Reply:
Object destructuring defaults are fantastic 😄 

https://github.com/Siteimprove/alfa/blob/7cfc5be90551302d4fc8beef34f4887f67d739de/packages/alfa-rules/test/common/evaluate.ts#L14-L19